Lottie.iOS/LOTAnimationView.cs

```diff
@@ -9,13 +9,15 @@ public partial class LOTAnimationView
         /// <summary>
         /// Asynchronously play the animation.
         /// </summary>
-        public Task<bool> PlayAsync()
+        public async Task<bool> PlayAsync()
         {
             var tcs = new TaskCompletionSource<bool>();
+            
+            this.CompletionBlock = animationFinished => tcs.SetResult(true);
+            
+            this.Play();
 
-            this.PlayWithCompletion((bool animationFinished) => tcs.SetResult(animationFinished));
-
-            return tcs.Task;
+            await tcs.Task;
```

Comment:
You still have to return the value?

Reply:
I don't, coz it's a Task and not a Task<bool>
But If you want we can change the signature async Task<bool>. But the previous PlayAsync did not return anything as well

Reply:
Sorry just recheck again. And you are right :). It is a task of bool. Gonna change that